automation/project/Settings.scala

```diff
@@ -1,6 +1,6 @@
 import Dependencies._
 import sbt.Keys._
-import sbt._
+import sbt.{Def, _}
```

Comment:
importing `Def` as well shouldn't be necessary, `_` means "everything in the package"